packages/visx-delaunay/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@visx/delaunay",
+  "version": "1.0.0",
+  "description": "visx delaunay",
+  "sideEffects": false,
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "esm"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/airbnb/visx.git"
+  },
+  "keywords": [
+    "visx",
+    "react",
+    "d3",
+    "visualizations",
+    "charts"
+  ],
+  "author": "@SheaJanke",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/airbnb/visx/issues"
+  },
+  "homepage": "https://github.com/airbnb/visx#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@types/d3-delaunay": "^6.0.1",
+    "@types/react": "*",
+    "classnames": "^2.3.1",
+    "d3-delaunay": "^6.0.2",
```

Comment:
This will be flagged as an [`ESM`-only dependency](https://github.com/airbnb/visx/blob/master/packages/visx-vendor/scripts/flagVendorRequirements.ts#L17), which means we actually need to add it to the new [`@visx/vendor`](https://github.com/airbnb/visx/pull/1716) package.

If things work as I hope, you should just be able to
- move the `d3-delaunay` + `@types/d3-delaunay` dependencies to the `@visx/vendor` `package.json`
- update the dependencies here to just be `@visx/vendor`
- update `d3-delaunay` **imports** in this package to `@visx/vendor/d3-delaunay`

Sorry for the extra steps, the ESM stuff is rough

Reply:
Thanks for the detailed instructions! It should be fixed now.